src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,25 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+
+    <hr />
+
+    <CommentList comments={post.comments} />
+  </div>
```

Comment:
Hi, I think you should add conditional statement here and launch CommentList module only if you have some comments.
Also `<hr />` tag is a line breaker between content that should only appear in case you don't have any comments. (Check the img below for reference)

![photo_2023-10-04_16-34-16](https://github.com/mate-academy/react_static-list-of-posts-js/assets/88306382/e85deac6-a721-4a7e-919e-1ebcfdd0c3e9)

Reply:
Thanks, your comment was helpful to me!